app/dcs/api/s3/s3-compatibility/page.md

```diff
@@ -11,104 +11,104 @@ The Storj S3-compatible Gateway supports a RESTful API that is compatible with t
+| GetObjectAcl                                | No      | No                                                           |                                                     |
+| GetObjectLegalHold                          | No      | No                                                           |                                                     |
+| GetObjectLockConfiguration                  | No      | No                                                           |                                                     |
+| GetObjectRetention                          | No      | No                                                           |                                                     |
+| GetObjectTagging                            | Full    |                                                              | Tags can be modified outside of tagging endpoints   |
+| GetObjectTorrent                            | No      | With significant effort, we could support this               |                                                     |
+| GetPublicAccessBlock                        | No      | No                                                           |                                                     |
+| HeadBucket                                  | Full    |                                                              |                                                     |
+| HeadObject                                  | Full    |                                                              |                                                     |
+| ListBucketAnalyticsConfigurations           | No      | No                                                           |                                                     |
+| ListBucketIntelligentTieringConfigurations  | No      | No                                                           |                                                     |
+| ListBucketInventoryConfigurations           | No      | No                                                           |                                                     |
+| ListBucketMetricsConfigurations             | No      | No                                                           |                                                     |
+| ListBuckets                                 | Full    |                                                              |                                                     |
+| ListMultipartUploads                        | Partial | Planned full. See https://github.com/storj/roadmap/issues/20 | See ListMultipartUploads section                    |
+| ListObjectVersions                          | No      | Planned. See https://github.com/storj/roadmap/issues/23      |                                                     |
+| ListObjects                                 | Partial | Planned full. See https://github.com/storj/roadmap/issues/20 | See ListObjects section                             |
```

Comment:
```suggestion
| ListObjects                                 | Partial | We may support lexicographical ordering in the future.  | See ListObjects section                             |
```

Reply:
I think we should get rid of references to https://github.com/storj/roadmap/issues/20